packages/webapp/src/members/components/member-chip.tsx

```diff
@@ -0,0 +1,146 @@
+            onClickChip={onClick}
+            contentComponent={
+                <MemberDetails member={member} onClick={onClick} />
+            }
+            actionComponent={<MemberChipNFTBadges member={member} />}
+        />
+    );
+};
+
+interface MemberDetailsProps {
+    member: MemberData;
+    onClick?: (e: React.MouseEvent) => void;
+}
+
+export const MemberDetails = ({ member, onClick }: MemberDetailsProps) => (
+    <div
+        onClick={onClick}
+        className="flex-1 flex flex-col justify-center group"
+    >
+        <p className="text-xs text-gray-500 font-light">
+            {member.createdAt === 0
+                ? "not an eden member"
+                : dayjs(member.createdAt).format("YYYY.MM.DD")}
+        </p>
+        <p className="group-hover:underline">{member.name}</p>
+        {member.account && (
+            <p className="text-xs text-gray-500 font-light">
+                @{member.account}
+            </p>
+        )}
+    </div>
+);
+
+export const MemberChipNFTBadges = ({ member }: { member: MemberData }) => (
+    <div className="absolute right-0 bottom-0 p-2.5 space-y-0.5">
+        <AuctionBadge member={member} />
+        <SaleBadge member={member} />
+        {!member.auctionData && !member.saleId && (
+            <AssetBadge member={member} />
+        )}
+    </div>
+);
+
+const AssetBadge = ({ member }: { member: MemberData }) => {
+    if (member.assetData) {
+        return (
+            <div
+                className="group flex justify-end items-center space-x-1"
+                onClick={(e) => {
+                    e.stopPropagation();
+                    openInNewTab(
+                        `${atomicAssets.hubUrl}/explorer/asset/${member?.assetData?.assetId}`
+                    );
+                }}
+            >
+                <p className="text-sm tracking-tight leading-none p-t-px group-hover:underline">
+                    NFT #{member.assetData.templateMint}
+                </p>
+            </div>
+        );
+    }
+    return <></>;
+};
+
+const AuctionBadge = ({ member }: { member: MemberData }) => {
+    if (member.auctionData) {
+        return (
+            <div
+                className="group flex justify-end items-center space-x-1"
+                onClick={(e) => {
+                    e.stopPropagation();
+                    openInNewTab(
+                        `${atomicAssets.hubUrl}/market/auction/${member?.auctionData?.auctionId}`
+                    );
+                }}
+            >
+                <FaGavel
+                    size={14}
+                    className="text-gray-600 group-hover:text-gray-800"
+                />
+                <p className="text-sm tracking-tight leading-none p-t-px group-hover:underline">
+                    {assetToString(member.auctionData.price, 2)} (#
+                    {member.assetData?.templateMint})
+                </p>
+            </div>
+        );
+    }
+
+    return <></>;
```

Comment:
Just a preference, but I'm a fan of having the intended code be unwrapped/primary. So
if (!member.auctionData) return <></>
//rest of code unrwapped

Reply:
Actually, that's usually my preference too. And less nesting. 👍

Reply:
Made that adjustment. Thanks!